### ISSUE TITLE
Change Name

### ISSUE THREAD

**Issue by wxtim** [campagnola/pyabc#12]
You should consider renaming this module because abc is widely recognised in python as abstract base class, making this module confusing and tricky to find.

**Comment by campagnola** [comment]
Good point; any suggestions? What did you search for before you found this?

**Comment by wxtim** [comment]
## Initial Respons
Honestly I can't remember what I was searching for. I wanted a parser for my own project. I'm surprised that there isn't more community for this really. I'll have a think...

## Response after some thought
I'm not feeling that inspired. I'd rather just use my branch to create guitar tabs and play guitar. 
Without more than two participants I'm not sure that it matters, unless the name prevents people finding it. If you did want to expand it (not an assumption I'm making) then asking for it to be added to the list at http://abcnotation.com/software might be handy. 

## Possible alt names
Some thoughts. None grab me. Space could be CamelCase :camel:  or hyphen
* pyabc parser
* pyabc music
* abc music
* pymusic abc
* pymusic parser
* py sheet music (parser)
* py score music (parser)

**Any name would ideally not get confused with libraries for interacting with mp3s either.**

If you do want to change it and think you might want to use pypi to package it later check https://pypi.org first. If you want to do this the name change becomes less optional, because the alternative is persuading someone else to give up the namespace https://pypi.org/project/pyabc/.

**Comment by jomimc** [comment]
I agree with this. Or perhaps instead integrating the code with other efforts to standardize code for parsing music notation. I think music21 is no longer maintained and at least one group is working on new parsing code.
